src/frontend/src/pages/AdminToolsPage/AdminToolsUserManagement.tsx

```diff
@@ -73,7 +75,10 @@ const AdminToolsUserMangaement: React.FC = () => {
       setHideSuccessLabel(false);
       setUser(null);
     } catch (e) {
-      alert(e);
+      if (e instanceof Error) {
+        alert(e.message);
```

Comment:
You don't need this line.

Reply:
I've removed alert(e) and alert(e.message), is that fine or should I add alert(e) back? Cause in another ticket in this epic I saw that alert(e) was removed.